### DISCUSSION TITLE
Why???? My GitHub Copilot suddenly stopped working, and I am an annual subscriber with the subscription not expiring until next year!!!

### DISCUSSION THREAD

**Discussion by yangdinghui** [community/community#129572] [Copilot Conversations]
### Select Topic Area

Question

### Body

```shell
2024-06-24 14:29:19.137 [info] [auth] Invalid copilot token: missing token: 403 
2024-06-24 14:29:19.160 [error] [auth] Extension activation failed: "Contact Support. You are currently logged in as yangdinghui."
2024-06-24 14:29:25.195 [error] [auth] Retrying auth
2024-06-24 14:29:30.475 [info] [auth] Invalid copilot token: missing token: 403 
2024-06-24 14:29:30.502 [info] [auth] Invalid copilot token: missing token: 403 
2024-06-24 14:29:30.528 [info] [auth] Invalid copilot token: missing token: 403 
2024-06-24 14:29:30.528 [error] [auth] Extension activation failed: "Contact Support. You are currently logged in as yangdinghui."
```

**Comment by duker-du** [comment]
same question

**Reply by duker-du** [reply]
Bro, try it: https://github.com/orgs/community/discussions/58529#discussioncomment-6370343

**Reply by yangdinghui** [reply]
It still can't be solved

**Comment by PerezChris99** [comment]
Check this out:
-The token might be missing or not provided correctly.
-There could be an issue with permissions or access rights associated with the token.
-The token might have expired or been revoked.
-This error message indicates that an extension (possibly related to Copilot or another service) could not be activated due to some issue related to the current user session.
-This could be related to authentication problems, user permissions, or a specific configuration issue with the extension.

**Comment by PerezChris99** [comment]
With the above-listed information, I think this could be highly helpful;
Check Authentication Tokens: Ensure that the Copilot token (and any other relevant tokens) are correctly configured, valid, and have the necessary permissions.

Verify User Session: Confirm that your user session is properly authenticated and has sufficient permissions to activate extensions or use services.

**Reply by yangdinghui** [reply]
It's still not working.This is my Copilot homepage.It's as if I've never subscribed, but the subscription doesn't expire until next year. 
![Github Copilot Index](https://github.com/community/community/assets/46334335/265bfdad-0337-4dad-91a1-e404dcda954d)

**Comment by PerezChris99** [comment]
i hope this works:
Sign out and sign back in to GitHub Copilot: This can often refresh the token and fix the issue.
Generate a new token: You can generate a new token from your GitHub account settings. Here's how:
Go to your GitHub settings https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features.
Click on "Developer settings".
Click on "Personal access tokens".
Click on "Generate new token".
Give your token a descriptive name and select the scopes that you need.
Click on "Generate token".
Copy the token and paste it into the GitHub Copilot settings.
Reinstall the GitHub Copilot extension: If the above steps don't work, you can try reinstalling the GitHub Copilot extension.
If you're still having trouble after trying these steps, you can contact GitHub support for help.

**Reply by yangdinghui** [reply]
ths,I am waiting for a response from GitHub Support.

**Comment by PerezChris99** [comment]
alright

**Comment by yangdinghui** [comment]
GitHub support resolution
